app/services/sensor/sensor.service.go

```diff
@@ -0,0 +1,86 @@
+package user
+
```

Comment:
This seems quite redundant no? Basically just redefining what was is sensor.repos.go. This could probably be merged with sensor.repos.go. I understand that design and architecture patterns are "best practices", but remember the number one rule - KISS. Also DRY. This is a bunch of extra code that is effectively equivalent to sensor.repos.go. You will even have overhead for having multiple linked calls.